pilot/pkg/model/push_context.go

```diff
@@ -2164,8 +2164,8 @@ func (ps *PushContext) ServiceAccounts(hostname host.Name, namespace string, por
 	}]
 }
 
-func (ps *PushContext) SupportsTunnel(ip string) bool {
-	infos, _ := ps.ambientIndex.PodInformation(sets.New(types.NamespacedName{Name: ip}))
+func (ps *PushContext) SupportsTunnel(ip string, clusterID string) bool {
+	infos, _ := ps.ambientIndex.WorkloadInfos(sets.New(types.NamespacedName{Name: ip, Namespace: clusterID}))
```

Comment:
Not sure I understand why you are assigning the Cluster ID to the namespace fields throughout this PR.
Can you please explain?

Reply:
To support multi-cluster pod info get